skiko/src/iosMain/kotlin/org/jetbrains/skiko/TextMenuArguments.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.skiko
+
+import org.jetbrains.skia.Rect
+
+class TextMenuArguments(
```

Comment:
It is subjective, but probably it is better to not to create a separate class for arguments of some function, because we can't figure out yet where it can be used.

Instead, we can just pass these as the actual arguments (as in Compose `TextToolbar`):
```
fun showTextMenu(
    targetRect: Rect, 
    onCopyRequested: (() -> Unit)?,
    onPasteRequested: (() -> Unit)?,
    onCutRequested: (() -> Unit)?,
    onSelectAllRequested: (() -> Unit)?,
)
```

Or extract something more meaningful with better defined purpose:
```
// TextMenu isn't appropriate name, because this can be used not only in menu, but for example when we perform a shortcut. Alternative names: TextActions, Text
interface TextManager {
    val copy: (() -> Unit)?,
    val paste: (() -> Unit)?
    val cut: (() -> Unit)?
    val selectAll: (() -> Unit)?
}

fun showTextMenu(targetRect: Rect, textManager: TextManager) {

}
```

But if you prefer the current code, we can keep it, because it is fine and my suggestions are not strong and probably subjective :)

Reply:
made interface
```Kotlin
interface TextActions {
    val copy: (() -> Unit)?
    val paste: (() -> Unit)?
    val cut: (() -> Unit)?
    val selectAll: (() -> Unit)?
}
```

Reply:
> @param textActions -

forgotten doc )

Reply:
Thanks!